src/NuGet.ProjectModel/LockFile.cs

```diff
@@ -61,5 +62,16 @@ public bool IsValidForPackageSpec(PackageSpec spec)
 
             return true;
         }
+
+        // DNU REFACTORING TODO: temp hack to make generated lockfile work with runtime lockfile validation
+        private static string RuntimeStyleLibraryRangeToString(LibraryRange libraryRange)
```

Comment:
`NuGet.LibraryModel.LibraryRange.ToString()` returns something like
`PackageName [MinVersion,)`
while the `LibraryRange.ToString()` in runtime returns
`PackageName >= MinVersion-*`
The hack here makes them consistent